src/test/java/io/strimzi/kafka/bridge/http/HttpCorsTests.java

```diff
@@ -111,7 +111,7 @@ public void testCorsNotEnabled(VertxTestContext context) {
                         client.request(HttpMethod.POST, 8080, "localhost", "/consumers/1/instances/1/subscription")
                                 .putHeader("Origin", "https://evil.io")
                                 .send(ar2 -> context.verify(() -> {
-                                    assertThat(ar2.result().statusCode(), is(405));
+                                    assertThat(ar2.result().statusCode(), is(400));
```

Comment:
Is this because of an empty subscription body?

Reply:
It seems that with Vert.x 3.9.3 they changed the status code as Bad Request.